tests/D2L.CodeStyle.Analyzers.Test/Specs/ImmutabilityAnalyzer.cs

```diff
@@ -769,17 +769,17 @@ public sealed class AnalyzedImmutableGenericClassGivenT<[ConditionallyImmutable.
 
 
 
-		static T /* MemberIsNotReadOnly(Field, m_field, AnalyzedImmutableGenericClassGivenT) */ m_field /**/;
-		static readonly T m_field;
+		static /* TypeParameterIsNotKnownToBeImmutable(T) */ T /**/ /* MemberIsNotReadOnly(Field, m_field, AnalyzedImmutableGenericClassGivenT) */ m_field /**/;

```

Comment:
The conditional parameters aren't added as immutable when verifying statics. They aren't safe to hold as static.

Reply:
Not that these updates are in the context of

```csharp
	[ConditionallyImmutable]
	public sealed class AnalyzedImmutableGenericClassGivenT<[ConditionallyImmutable.OnlyIf] T, U> {
```